src/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ find_package(Boost ${requiredBoostVersion} REQUIRED)
 find_package(LibtorrentRasterbar ${requiredLibtorrentVersion} REQUIRED)
 find_package(OpenSSL ${requiredOpensslVersion} REQUIRED)
 
-if (Boost_VERSION VERSION_LESS 106000)
```

Comment:
Why this one doesn't work?

Reply:
idk but I noticed following defination was still added on my builds which are configured with boost 1.72, I copied new condition from here https://github.com/Kitware/CMake/blob/f74a963021b1ca26c7145f53a622183522108bdf/Modules/FindBoost.cmake#L753

you can find many such conditions on same file done "new" way